src/util/util.spec.ts

```diff
@@ -1,5 +1,15 @@
 import {toInteger, toString, getValueInRange, isInteger, isString, hasClassName} from './util';
 
+afterAll(() => {
```

Comment:
What is the reasoning behind putting this global `afterAll` here inside `util/util.spec.ts` ?